great_expectations/checkpoint/checkpoint.py

```diff
@@ -153,7 +153,7 @@ def run(  # noqa: C901 - complexity 19
         profilers: Optional[List[dict]] = None,
         run_id: Optional[Union[str, RunIdentifier]] = None,
         run_name: Optional[str] = None,
-        run_time: Optional[Union[str, datetime.datetime]] = None,
+        run_time: Optional[datetime.datetime] = None,
```

Comment:
@Kilo59 Would this not lead to type checking errors?..

Reply:
It didn't cause any mypy errors. That doesn't mean it won't ever, just that it didn't based on our current type-checking coverage.